native/cocos/renderer/gfx-metal/MTLUtils.mm

```diff
@@ -953,6 +953,8 @@ void main() {
 #endif
     options.emulate_subgroups = true;
     options.pad_fragment_output_components = true;
+    // fully support
+    options.set_msl_version(2, 0, 0);
```

Comment:
What devices and what iOS version can use it?

Reply:
ios11+ macos10.14+